setup.py

```diff
@@ -64,9 +64,9 @@
 
 base_dir = os.path.dirname(os.path.abspath(__file__))
 
-_date = '20240726'
+_date = '20240801'
 _libtpu_version = f'0.1.dev{_date}'
-_libtpu_storage_path = f'https://storage.googleapis.com/libtpu-nightly-releases/wheels/libtpu-nightly/libtpu_nightly-{_libtpu_version}+default-py3-none-any.whl'
+_libtpu_storage_path = f'https://storage.googleapis.com/libtpu-nightly-releases/wheels/libtpu-nightly/libtpu_nightly-{_libtpu_version}+nightly-py3-none-any.whl'
 _jax_version = f'0.4.31.dev{_date}'
```

Comment:
Please double check the JAX minor version against their index: https://storage.googleapis.com/jax-releases/jax_nightly_releases.html

Reply:
Thanks for catching it! Updated to `0.4.32`